src/identities/identities.controller.ts

```diff
@@ -465,4 +466,51 @@ export class IdentitiesController {
     const identity = await this.identitiesService.createMockCdd(params);
     return createIdentityModel(identity);
   }
+
+  @ApiTags('claims')
+  @ApiOperation({
+    summary: 'Retrieve the list of InvestorUniqueness claims for a target Identity',
+    description:
+      'This endpoint will provide a list of all the InvestorUniquenessClaims made about an Identity',
+  })
+  @ApiParam({
+    name: 'did',
+    description: 'The DID of the Identity for which to fetch InvestorUniquenessClaims',
+    type: 'string',
+    example: '0x0600000000000000000000000000000000000000000000000000000000000000',
+  })
+  @ApiQuery({
+    name: 'includeExpired',
+    description:
+      'Indicates whether to include expired InvestorUniquenessClaims or not. Defaults to true',
+    type: 'boolean',
+    required: false,
+  })
+  @ApiArrayResponse(InvestorUniquenessModel, {
+    description: 'List of InvestorUniquenessClaims for the given DID',
+    paginated: false,
+  })
+  @Get(':did/investor-uniqueness-claims')
+  async getInvestorUniquenessClaims(
+    @Param() { did }: DidDto,
+    @Query() { includeExpired }: IncludeExpiredFilterDto
+  ): Promise<InvestorUniquenessModel[]> {
+    const investorUniquenessClaims = await this.claimsService.getInvestorUniquenessClaims(
+      did,
+      includeExpired
+    );
+
+    const results = investorUniquenessClaims.map(
+      ({ issuedAt, expiry, claim, target, issuer }) =>
+        new InvestorUniquenessModel({
+          issuedAt,
+          expiry,
+          claim,
+          target,
+          issuer,
+        })
+    );
+
+    return results;
```

Comment:
```suggestion
    return { results };
```
Based on your usage of ApiArrayResponse, I think you want to return the above as resultset